shared/src/main/scala/mlscript/Typer.scala

```diff
@@ -62,15 +62,52 @@ class Typer(var dbg: Boolean, var verbose: Bool, var explainErrors: Bool, val ne
       env: MutMap[Str, TypeInfo],
       mthEnv: MutMap[(Str, Str) \/ (Opt[Str], Str), MethodType],
       lvl: Int,
+      qenv: MutMap[Str, SkolemTag], // * SkolemTag for variables in quasiquotes
```

Comment:
This name is terrible. Please try at least a little to find a more descriptive one.